src/ec.ts

```diff
@@ -0,0 +1,321 @@
+import { NativeQuickCrypto } from './NativeQuickCrypto/NativeQuickCrypto';
+import { bufferLikeToArrayBuffer, type BufferLike } from './Utils';
+import {
+  type ImportFormat,
+  type SubtleAlgorithm,
+  type KeyUsage,
+  kNamedCurveAliases,
+  type NamedCurve,
+  PublicKeyObject,
+  KWebCryptoKeyFormat,
+  CryptoKey,
+} from './keys';
+
+// const {
+//   ArrayPrototypeIncludes,
+//   ObjectKeys,
+//   SafeSet,
+// } = primordials;
+
+// const {
+//   ECKeyExportJob,
+//   KeyObjectHandle,
+//   SignJob,
+//   kCryptoJobAsync,
+//   kKeyTypePrivate,
+//   kSignJobModeSign,
+//   kSignJobModeVerify,
+//   kSigEncP1363,
+// } = internalBinding('crypto');
+
+// const {
+//   getUsagesUnion,
+//   hasAnyNotIn,
+//   jobPromise,
+//   normalizeHashName,
+//   validateKeyOps,
+//   kHandle,
+//   kKeyObject,
+//   kNamedCurveAliases,
+// } = require('internal/crypto/util');
+
+// const {
+//   lazyDOMException,
+//   promisify,
+// } = require('internal/util');
+
+// const {
+//   generateKeyPair: _generateKeyPair,
+// } = require('internal/crypto/keygen');
+
+// const {
+//   InternalCryptoKey,
+//   PrivateKeyObject,
+//   PublicKeyObject,
+//   createPrivateKey,
+//   createPublicKey,
+// } = require('internal/crypto/keys');
+
+// const generateKeyPair = promisify(_generateKeyPair);
+
+// function verifyAcceptableEcKeyUse(name, isPublic, usages) {
```

Comment:
Do we need all os those comments?

Reply:
I'm slowly uncommenting some of them (most🤞)

Reply:
It's just the remaining source code for the rest of the web crypto implementation. It will come back as the rest of the API is filled.